indico/modules/events/sessions/ical.py

```diff
@@ -0,0 +1,52 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2021 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from icalendar.cal import Calendar
+from werkzeug.urls import url_parse
+
+from indico.core.config import config
+from indico.modules.events.ical import generate_basic_component
+from indico.web.flask.util import url_for
+
+
+def generate_session_component(session, related_to_uid=None):
+    """Generates an Event iCalendar component from an Indico Session."""
```

Comment:
```suggestion
    """Generate an Event iCalendar component from an Indico Session."""
```

> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".

We fixed many of docstrings in #4649 to follow [PEP-257](https://www.python.org/dev/peps/pep-0257/) so let's keep new ones consistent :)